### DISCUSSION TITLE
AutoConnect statistics screen portal

### DISCUSSION THREAD

**Discussion by spike322** [Hieromon/AutoConnect#365] [Q&A] [answered]
Whenever I connect to the esp access point, if I try to reach the AutoConnect portal at http://172.217.28.1/_ac it doesn't show me the "website", but a strange string, which I have reported below:

```
!OTP tltteAtCnetsaitctlfn-aiyHleiaAilsn-ei;otsz:6x-stx-ieajs:0%-ekttx-ieajs:0%-o-s-otsotiggasae-ektfn-mohn:nilae}oymri:;adn:}bs-ae{agn02p 2x.aepnl*lbl:o(bn)dslyiln-lc;it:.e;etainrgt.aepnl*.ls{it:uofn-ie09mmri-et1p;etainlf}nu{mzapaac:oe-ektapaac:oefn-ie09mmri:p uo.a{iiiiyclas}lptre{iiiiyvsbe.a:agt.vra{pct:.;rniin03}lptre mdlbto{pct:;rniin03}oelptp0lf:;it:0%hih:0%psto:ie;pct:;akrud#0;-ne:00.oa_utnbre-ais1p;akrud#603clr#ffcpdig2p 0xtx-lg:etrtx-eoainnn;etrsaig1xfn-egtbl;ipa:nieboktp4%lf:0;it:0;oiinfxdoaiy0zidx10}al{odrclas:olpebre-pcn:;odr1xsld#d;oo:44bcgon-oo:ffmri-otm2p}al.notbeif>fo,al.notedwdh10;odrclr#CDDtbeif>ha{akrudclr#CDDtbeif>ha>rt{oo:fft,hpdig1p 2xtedbcgon-oo:fff;odrbto:p oi ddtedt>hfn-egt40tx-lg:ettotbre-o:p oi ddtoytoyt:t-hl(d)bcgon-oo:fftoyt>dtott>dfn-egt30fn-ie.8mtoyt:t-hl(vn{akrudclr#777tbeif bd>rnhcidee)bcgon-oo:EFF}l-ie{it:0%psto:ie;o:;et0zidx10;o-hdw01x3xrb(,0 ,01) p p ga0 ,0 .4}l-ugrsa,l-ugrsa:bfr,l-ugrsa:atrdslybokhih:p;it:6xtasto:.ses}l-bcekd.bmn i.bbre pnbcgon-oo:rnprn}l-bcekd.bmn i.bbre pn:eoe.bc:hce~l-eul l-ugrsa:atrmri-o:}l-edrdslyfe;lxdrcinrwjsiycnetsaebtenainiescne;egt5p}l-eurgt.bbre{agnlf:uo.bbadfn-ie16mpdig1p 4x1p 4x.bmn{i-egt5p;rniin06 aewdh10}l-aiaindslyfe;lxdrcinclm;itsyenn;adn-et0mri:}l-eua.bie {etdcrto:oeclrihrtcro:one}l-tmhih:8x.bie {adn:8x2p 8x2p;ipa:lc}l-ugrpdig1p 4x1p 4xpsto:eaiecro:one}l-ugrsa:bfr,l-ugrsa:atrcnet';oiinaslt}l-ugrsa:bfr{agntp-p}l-ugrsa:atrmri-o:p}l-bdslynn}l-bnt:hce).bmn{vrlwhde;egt5p}l-bcekd.bmn{rniinhih .ses;egt10hoefo:uo.rponpsto:eaiehih:uomnhih:8x.rponhvru{oiinrltv;ipa:lc;i-it:0%.rpona:fe{oiinaslt;otn:'rgt1p;o:5xbre-it:p p ;odrclrtasaetbre-tl:oi}dodw>ldslybokoefo-:idnls-tl:oepdig0.rponu l-tmmnwdh10;egt2p;adn:p 0x5x4p}dodw>l.bie {i-egt2p;iehih:9xpdig0@ei cenad(i-it:6p).bnvgto{lxfo:o;utf-otn:lxed}l-ugrdslynn;.bc:o(cekd~l-euoefo:iil;.bc:hce~l-euhih:8x}l-eu.bie{odrtp0}l-eurgt.bhae{agnrgtat;.rponhih:8x}dodw:oe>lpsto:bouelf:;o:8xpdig0}dodw>ldslynn;.rponu l-tmpdig5x1p;.rponu l-tmawiesaenwa;}l-bcekd.bmn l-ugrdlpnsa:bfr{rnfr:oae25e).bc:hce+l-eu.bbre-bsi pn:fe{rnfr:oae-2dg}l-eumtra,l-eumtra dodw lbcgon-oo:233;oo:ff.bmn-aeil.cie.bmn-aeil.bie:oe{akrudclr#77f.bmn-aeil.bbre pn.bmn-aeil.bbre pn:eoe.bmn-aeil.bbre pn:fe{akrudclr#f}/tl>/edhae d"b ls=l-ie"dvcas"bmn bmn-ih bmn-aeil>u ls=l-aiain>l ls=l-edr>ahe=/a"cas"bbad>uoonc/ae>/il ls=l-tm>ahe=/a/pn>pnSIs/>/il ls=l-tm d"ee"< rf"rl"Rst.l ls=l-tm>ahe=/>OE/>/i/i>dvcas"a"i=rl"< rf"rst ls=oelp>/>dvcas"oa_utn>h>ahe=/a/ee"cas"oa_utn>EE/i>/i>/edrtbecas"no tl=bre:oe"t>t>salse oncin/dt>t>oe/dt>t>P/d/rt>I ne)/dt>t>untmst>I ne)/dt>t>otPIt>7.1.81/dt>t>PMC/dt>t>T At>8F:4C:F9/rt>t>Bt>//r/rt>t>ls ie/dt>t>remmrt>84/r/tlÿ
```

I don't know why I'm seeing this, I've just followed the simple example sketch reported in here: https://hieromon.github.io/AutoConnect/gettingstarted.html

**Comment by Hieromon** [comment]
The response you posted is not from AutoConnect. Is it possible that the higher router of the WiFi access point you connected to is redirecting access to `172.217.28.1/_ac` to a different URL?
You can use a network traffic diagnostic tool such as Chrome DevTools to trace whether HTTP access to `172.217.28.1/_ac` is reaching the ESP module via 0the correct route.

The screenshot below is a trace analysis of a request from a Chrome browser to `172.217.28.1/_ac` using Chrome Dev Tools. An access to `172.217.28.1/_ac` is complete with HTTP code 200 without being replaced by other outbound requests.
So, verify your network environment and constraints in the same way.

This topic has nothing to do with the library feature and will be migrated to the discussions.

<img src="https://user-images.githubusercontent.com/12591771/120418312-3f0b8980-c39b-11eb-952d-37e4bf9cb2fd.png" width="480px">

**Reply by HedgeTheHog** [reply]
Hi, I think I have same problem as @spike322 , here is a screenshot. I tried to connect from two devices 
![image](https://user-images.githubusercontent.com/12202527/120534396-0f33a480-c3e2-11eb-97ab-bcefd4eb5fa4.png)
 Edit: I am using nodemcu v1

**Reply by lucasheld** [reply]
I also have the same problem. This only occurs with esp8266 board version 3.0.0. After downgrade to 2.7.4, AutoConnect works as expected again.

**Reply by Hieromon** [reply]
@lucasheld Thank you your report.
I found out why I can't reproduce this issue in my environment. AutoConnect does not yet support ESP8266 core 3.0.0.

**Comment by Hieromon** [comment, marked as answer]
Assuming this issue is incompatible with ESP8266 arduino core 3.0.0, you will have to wait for AutoConnect to support the ESP8266 arduino 3.0.0. Or use an older generation core.
Same issue already posted: #361 

AutoConnect plans to support ESP8266 Arduino core 3.0.0 in the next update turn as v1.3.0. But you can evaluate ESP8266 core 3.0.0 with AutoConnect v1.3.0 development version staged on the branch of the [enhance/v130](https://github.com/Hieromon/AutoConnect/tree/enhance/v130).
At that time, you also need to update [PageBuilder](https://github.com/Hieromon/PageBuilder) to [v1.5.0](https://github.com/Hieromon/PageBuilder/tree/enhance/v150). PageBuilder is also not compatible currently with ESP8266 arduino core 3.0.0, and I'm working in parallel with the enhancement of AutoConnect. The development version of PageBuilder v1.5.0 (which is the version that supports ESP8266 arduino core 3.0.0) is staged on [this branch](https://github.com/Hieromon/PageBuilder/tree/enhance/v150).
